### PR TITLE
EXLM 1303 Callout Block UE Changes

### PR DESCRIPTION
**Pull request by devisree29**
Updated Component Model Definition and Filters for Callout Block 

Jira ID:https://jira.corp.adobe.com/browse/EXLM-1303

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/en/articles/experience-manager/sample-article
- After: https://feature-exlm-1303-callout--exlm--adobe-experience-league.hlx.live/en/articles/experience-manager/sample-article

image

### CODE REVIEW

File: component-models.json [hunk 1]
```diff
@@ -2470,5 +2470,60 @@
...
+  {
+    "id": "callout",
+    "fields": [
+      {
+        "component": "richtext",
+        "valueType": "string",
+        "name": "first-callout-text",
+        "value": "",
+        "label": "First Callout Text",
+        "description": "Sets the text"
```
Comment (ruhisingh1) [inline component-models.json:2483]: Can we provide more specific description like - "Sets the text for first callout block." ?

File: component-definition.json [hunk 2]
```diff
@@ -683,6 +683,22 @@
...
+          "title": "Callout",
+          "id": "callout",
+          "plugins": {
+            "xwalk": {
+              "page": {
+                "resourceType": "core/franklin/components/block/v1/block",
+                "template": {
+                  "name": "Callout",
+                  "model": "callout",
+                  "title": "callout"
```
Comment (ruhisingh1) [inline component-definition.json:697]: "title" property is not required. We can remove this.

Comment (devisree29) [reply component-definition.json]: Updated

File: component-models.json (hunk 1 above)
Comment (devisree29) [reply component-models.json]: Updated

**Review by ruhisingh1 — APPROVED** [review]

**Pull request merged**